### PR TITLE
chore: update placeholder

### LINKED ISSUE #13618: [Bug]: Default values placeholder on phone widget is incorrect
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Placeholders for `default text` property for phone input widgets are incorrect. It has `John Doe` as placeholder even though it don't accept string type input

### Steps To Reproduce

image

### Public Sample App

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by ganganimaulik**
Fixes #13618

## Type of change
- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: app/client/src/widgets/PhoneInputWidget/widget/index.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ class PhoneInputWidget extends BaseInputWidget<
               propertyName: "defaultText",
               label: "Default Text",
               controlType: "INPUT_TEXT",
-              placeholderText: "John Doe",
+              placeholderText: "000 0000",
```
[the remark this answers is not in the document]
Comment (ganganimaulik) [reply app/client/src/widgets/PhoneInputWidget/widget/index.tsx:109]: just took it from                
     example: `000 0000`,

[the remark this answers is not in the document]
Comment (sbalaji1192) [reply app/client/src/widgets/PhoneInputWidget/widget/index.tsx:109]: @somangshu @dilippitchika Do we have a verdict here? Shall we go ahead with the default US placeholder for example and placeholders?

[the remark this answers is not in the document]
Comment (somangshu) [reply app/client/src/widgets/PhoneInputWidget/widget/index.tsx:109]: Yeah, thats my best bet too. EX > (555) 555-1234

@dilippitchika @sbalaji1192 shall we go forward with this?

[the remark this answers is not in the document]
Comment (sbalaji1192) [reply app/client/src/widgets/PhoneInputWidget/widget/index.tsx:109]: @somangshu `(555) 555-1234` could be someone's phone number (not sure though). Let's go ahead with `(000) 000-000`?

[the remark this answers is not in the document]
Comment (somangshu) [reply app/client/src/widgets/PhoneInputWidget/widget/index.tsx:109]: Copied it from google, But yeah lets go with more generic.

@ganganimaulik can you make the change here?

Apologies for the decision paralysis here 🙃

[the remark this answers is not in the document]
Comment (ganganimaulik) [reply app/client/src/widgets/PhoneInputWidget/widget/index.tsx:109]: no problem, should I also update example or only placeholder? @somangshu

[the remark this answers is not in the document]
Comment (sbalaji1192) [reply app/client/src/widgets/PhoneInputWidget/widget/index.tsx]: @ganganimaulik Yes please.

**Review by somangshu — APPROVED** [review]

**Review by somangshu — APPROVED** [review]
LGTM

**Pull request merged**